docs_src/_layouts/people.html

```diff
@@ -5,17 +5,17 @@
 <!-- check out if there is any member with the intended role and create a boolean variable for it -->
 
 {% for member in site.members %}
-    {% if member.role == 'phd_student' %}
-        {% assign phd_student_exists = true %}
-    {% elsif member.role == 'master_student' %}
-        {% assign master_student_exists = true %}
-    {% elsif member.role == 'undergrad_student' %}
-        {% assign undergrad_student_exists = true %}
-    {% elsif member.role == 'highschool_student' %}
-        {% assign highschool_student_exists = true %}
-    {% elsif member.role == 'REU' %}
-        {% assign REU_exists = true %}
-    {% endif %}
+{% if member.role == 'phd_student' %}
+{% assign phd_student_exists = true %}
+{% elsif member.role == 'master_student' %}
+{% assign master_student_exists = true %}
+{% elsif member.role == 'undergrad_student' %}
+{% assign undergrad_student_exists = true %}
+{% elsif member.role == 'highschool_student' %}
+{% assign highschool_student_exists = true %}
+{% elsif member.role == 'alumni' %}
```

Comment:
Alumni:= gender-neutral and plural, Alumnus:=male gender and singular, Alumna: female gender and singular. =)